cli/flox/src/commands/activate.rs

```diff
@@ -126,7 +126,11 @@ pub struct Activate {
 
 impl Activate {
     pub async fn handle(self, mut config: Config, flox: Flox) -> Result<()> {
-        subcommand_metric!("activate");
+        environment_subcommand_metric!(
+            "activate",
+            self.environment,
+            start_services = self.start_services
+        );
```

Comment:
Should we add this for either of
```
subcommand_metric!("activate#version", lockfile_version = lockfile_version);
```
or
```
        subcommand_metric!(
            "install",
            "failed_packages" = Install::format_packages_for_tracing(packages)
        );
```

I don't fully understand how those are used

Reply:
Parsing error: I'm not sure what "this" is in this comment.

Reply:
Should we use `environment_subcommand_metric` instead of `subcommand_metric` so that we record `--remote` for
```
subcommand_metric!("activate#version", lockfile_version = lockfile_version);
```

and

```
        subcommand_metric!(
            "install",
            "failed_packages" = Install::format_packages_for_tracing(packages)
        );
```